package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gcloud-storage-upload",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
You should execute the release task after feature additions. There should be a plain release commit.